radio/src/gui/colorlcd/model_mixer_scripts.cpp

```diff
@@ -75,17 +75,26 @@ class ScriptEditWindow : public Page {
       FormGridLayout grid;
       grid.spacer(PAGE_PADDING);
 
-      ScriptData* sd = &(g_model.scriptsData[idx]);
+      ScriptData* const sd = &(g_model.scriptsData[idx]);
 
+      // ***: special case of deleted entry resused here
+      // should first delete all entries in inputs/outputs to prevent stale display
+      if (sd->file[0] == '\0') {
```

Comment:
It might make more sense to trigger the reload on "delete", and then checking the interpreter state before editing the params rather than this funky hack.

Reply:
Yes, changed that (see comments above)